LLMonFHIR/FHIRInterpretation/MultipleResources/MultipleResourceChatViewModel.swift

```diff
@@ -0,0 +1,105 @@
+//
+// This source file is part of the Stanford Spezi project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SpeziChat
+import SpeziLLM
+import SwiftUI
+
+/// View model for the MultipleResourcesChatView.
+///
+/// This view model coordinates between the UI and the FHIRMultipleResourceInterpreter.
+/// It provides UI-specific computed properties and methods while delegating
+/// LLM operations and persistence to the underlying interpreter.
+@MainActor
+final class MultipleResourcesChatViewModel: ObservableObject {
```

Comment:
I think we should generally aim to use `@Observable` here and in other places in the app. Seems like none of these properties are `@Publishable` but I would assume that we should actually listen to changes in `textToSpeech`.